Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -454,7 +452,10 @@ - (void)disabled_testRetrieveUserInfo {
 /// The login queue argument should be respected.
 - (void)testLoginQueueForSuccessfulLogin {
     // Make global queue
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunguarded-availability"
```

Comment:
Why not switch to using `DISPATCH_QUEUE_PRIORITY_DEFAULT` instead of a constant that isn't available on some versions of macOS that this test target builds for?

Reply:
I think they're all gated by the same availability check. Let me try it...

Reply:
I don't think so… the QoS stuff is newer than dispatch itself.

Reply:
The `QOS` constants are new, the `DISPATCH_QUEUE_PRIORITY` ones aren't.